### DISCUSSION TITLE
Citation for Scoring Metric

### DISCUSSION THREAD

**Discussion by krn-hov** [MilesCranmer/PySR#326] [General]
Hi! I remember seeing somewhere that there were separate citations for the pySR software and, I think, the scoring metric. I can't seem to find these separate citations now, could someone point me to them?

**Comment by MilesCranmer** [comment]
The paper you are thinking of is https://arxiv.org/abs/2006.11287

(maybe I should add it back to a combined citation file?)

cheers,
Miles
